### DISCUSSION TITLE
关于Fakeip无法ping

### DISCUSSION THREAD

**Discussion by taoersvip** [kkkgo/PaoPaoDNS#196] [问答和帮助]
因为局域网设备会不停地icmp--Google.com，以验证是否联网正常，解析出来是fakeip，无法正常发送请求，而与透明网关之间，造成回环，请问如何设置像这样的个别域名返回真实ip，并通过指定网关让icmp服务正常！谢谢！
Ros+paopaodns+singbox

**Comment by kkkgo** [comment]
你要返回真实IP可以直接加入/data/force_dnscrypt_list.txt。当然返回真实IP也不等于你一定能ping通。
至于你的fakeip不能ping其实是你的singbox网关处理的问题，具体怎么处理可能需要你看singbox的文档，如果你用paopaogateway应该是可以ping通的。

**Reply by taoersvip** [reply]
谢谢，明白了，paopao大佬
